packages/db/src/lib/granule.ts

```diff
@@ -354,3 +360,61 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
+ *
+ * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+ * @param {Array<ApiGranuleRecord>} [granules] - updated ApiGranule records
```

Comment:
Are these going to be full ApiGranuleRecords, or is a partial allowed?    This feels like you might want to consider ApiGranule instead, or else enforce a new type that has update fields if they're required.